cypress/support/pages/admin/pageAdministration.js

```diff
@@ -1,424 +1,473 @@
+        cy.get(Management.#migrationEndButton)
+            .should('be.enabled')
+    }
+
+    clickEndMigrationButton() {
+        cy.get(Management.#migrationEndButton)
+            .click()
+    }
+
+    clickEndMigrationConfirmationCheckbox() {
+        cy.get(Management.#endMigrationConfirmationCheckbox)
+            .check( {force: true} )
+    }
+
+    clickEndMigrationConfirmationButton() {
+        cy.get(Management.#endMigrationConfirmationButton)
+            .click()
+    }
+
+    seeEndMigrationInformationTitle() {
+        cy.get(Management.#endMigrationInformationTitle)
+            .should('be.visible');
+    }
+
+    seeEndMigrationInformationText() {
+        cy.get(Management.#endMigrationInformationText)
+            .should('be.visible');
+    }
+
+    checkEndMigrationInfoLink() {
+        cy.get(Management.#endMigrationTextBlogLink)
+            .should("have.attr", "href", "https://blog.niedersachsen.cloud/umzug")
+    }
+
+    seeEndMigrationConfirmationCheckboxIsUnchecked() {
+        cy.get(Management.#endMigrationConfirmationCheckbox)
+            .should('not.be.checked');
+    }
+
+    seeEndMigrationAbortButton(){
+        cy.get(Management.#endMigrationAbortButton)
+            .should('be.visible');
+    }
+
+    seeEndMigrationConfirmationButtonIsDisabled(){
+        cy.get(Management.#endMigrationConfirmationButton)
+            .should('be.disabled')
+    }
+
+    seeEndMigrationConfirmationButtonIsEnabled(){
+        cy.get(Management.#endMigrationConfirmationButton)
+            .should('be.enabled')
+    }
+
+    seeMigrationMandatorySwitch(){
+        cy.get(Management.#migrationMandatorySwitch)
+            .should('not.be.checked');
+    }
+
+    seeSyncDuringMigrationSwitchIsNotChecked(){
+        cy.get(Management.#enableSyncDuringMigrationSwitch)
+            .should('not.be.checked')
+    }
+
+    seeSyncDuringMigrationSwitchIsChecked(){
+        cy.get(Management.#enableSyncDuringMigrationSwitch)
+            .and('be.checked')
+    }
+
+    toggleMigrationMandatorySwitch(){
+        cy.get(Management.#migrationMandatorySwitch)
+            .click();
+    }
+
+    toggleSyncDuringMigrationSwitch(){
+        cy.get(Management.#enableSyncDuringMigrationSwitch)
+            .click();
+    }
+
+    seeMigrationFinishedTimestamp(){
+        cy.get(Management.#migrationFinishedTimestamp)
+            .and('be.visible')
+    }
+
+    clickExternalToolsPanel(){
+        cy.get(Management.#externalToolsPanel)
+            .click()
+    }
+
+    clickDeleteExternalToolButton() {
+        cy.get(Management.#deleteExternalToolButton)
+            .first()
+            .click()
+    }
+
+    clickCancelExternalToolDeletionButton() {
+        cy.get(Management.#cancelExternalToolDeletionButton)
+            .click()
+    }
+
+    seeExternalToolTable() {
+        cy.get(Management.#externalToolsTable)
+            .should('be.visible')
+    }
+
+    seeExternalToolDeletionDialog() {
```

Comment:
i think the dialog always exists in the DOM. Maybe a visible would be better here

Reply:
removed